.travis.yml

```diff
@@ -5,7 +5,8 @@ script: ./b clean build
 sudo: false
 
 jdk:
-  - oraclejdk8
+  - oraclejdk11
```

Comment:
Travis grumbles about 8, so figured we might as well have the LTS and the latest 6-mo release.